internal/docker/container.go

```diff
@@ -36,6 +37,7 @@ type ContainerRunner struct {
 	ShowConsoleLog    bool
 	JobReader         job.Reader
 	ArtfactDownloader download.ArtifactDownloader
+	SlackReporter     report.Reporter
```

Comment:
Now that the slack reporter is implementing the `report.Reporter` interface, you can pass it along with the other reporters. That way, when the the logic of calling the reporters is executed (https://github.com/saucelabs/saucectl/pull/418/files#diff-4a9412a6fc600dd10c8d4eb6f62f7dfba9ad7612975616334f8613fb9c559c69R371), it'll also be applied to the slack reporter.

Reply:
I refactored this part.